### DISCUSSION TITLE
Isar Database Data Persists After Reinstalling Flutter App

### DISCUSSION THREAD

**Discussion by ApplicationPaker** [isar/isar#1593] [Help] [answered]
**Issue:**
I have encountered an issue with Isar database where data persists even after reinstalling the Flutter app. Despite reinstalling the app, old data stored in the Isar database reappears upon relaunching the app. This behavior raises concerns about data privacy and app reliability.

**Comment by Jtplouffe** [comment, marked as answer]
This might be due to android's auto backup feature.
You could try setting `allowBackup` to false in the Android manifest file.

**Reply by ApplicationPaker** [reply]
Thanks
